### PR TITLE
Repairing the Gameplay logic for Happy Path 

### PR DESCRIPTION
**Pull request by artofmayhem**
This PR is to reset the Happy Path logic/ routes of working gameplay. 

Files Changed: 

Index.js: replaced component imports and routes
Gamemode.less: replaced styling code for Gamemode path components

Folders Added: 
DrawingSub
     - DrawingSubContainer.js
     - index.js
     - RenderingDrawingSub.js

WritingSub
     - index.js
     - RenderingWritingSub.js
     - WritingSubContainer.js

This fixes routing. If you notice any of your code missing I rebased just before class started so I'm pretty sure I can find it and restore it. 

PR/ Product Demo Video: https://youtu.be/akHsYm3v0jU

### CODE REVIEW

**Review by rhiannonstanford — APPROVED** [review]
Thank you for this fix. Nice work

File: src/components/pages/WritingSub/RenderWritingSub.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+};
+
+export default connect(
+  state => ({
+    tasks: state.tasks,
+  }),
+  {
+    setHasWritten: tasks.setHasWritten,
+  }
+)(RenderWritingSub);
```
Comment (MichaelMcFaddenjr) [inline src/components/pages/WritingSub/RenderWritingSub.js:80]: Great clean code

File: src/index.js [hunk 2]
```diff
@@ -41,20 +42,22 @@ import { NotFoundPage } from './components/pages/NotFound';
...
 
-// Gameification Components
-import { GameificationMain } from './components/pages/Gameification';
+// GamePlay Components
+import { GamePlayMain } from './components/pages/GamePlay/index';
+import { PlayAgain } from './components/pages/PlayAgain';
 import { PointShare } from './components/pages/PointShare';
 import { MatchUp } from './components/pages/MatchUp';
 import { VotingPage } from './components/pages/VotingPage';
-import { PlayAgain } from './components/pages/PlayAgain';
```
Comment (MichaelMcFaddenjr) [inline src/index.js:57]: Good cleanup

**Review by MichaelMcFaddenjr — APPROVED** [review]

**Review by Yonathan-Admasu728 — COMMENTED** [review]
Great fixes. Nice work.

**Review by Yonathan-Admasu728 — COMMENTED** [review]
Great fixes across all 9 files. Nice work.

**Pull request merged**